---
layout: about
title:  CV
---

# Caitlin Guccione
## Bioinformatics Ph.D. Student   

> [cguccion@ucsd.edu](mailto:cguccion@ucsd.edu)  
> (630) 336-3386  

**Objective**
I am a Ph.D. graduate student with a projected summer 2024 graduation looking for a computational biology
role in industry. My passion is harnessing the power of big data with my computer science and mathematics
expertise to tackle complex biological challenges, particularly in the field of oncology.

**Interests** : Bioinformatics, Computional Biology, Oncology, Cancer Microbiome 

## Education
### Ph.D. Bioinformatics & Systems Biology  
**University of California, San Diego**  
*September 2020 - Present*
- **Thesis** : Cancer Microbiome + Microbial Modeling
- **Advisors** : [Dr. Rob Knight](https://knightlab.ucsd.edu)  & [Dr. Kit Curtius](https://qcclab.com) 

### M.S. Applied Mathematics
**University of Rhode Island**  
*September 2018 - August 2020*
- **Final Project** : Stochastic Models with Applications to Genetics, Cancer, AIDS  & Biomedical Systems 

### B.S. Applied Mathematics & B.A. Computer Science
**University of Rhode Island**   
*September 2016 - May 2020*
- **Minor** : Biological Sciences
- **Honors** : magna cum laude, University of Rhode Island Honors Program
- **Extracurriculars** : [NCAA DI Student-Athlete](https://gorhody.com/sports/womens-swimming-and-diving/roster/caitlin-guccione/5710), School Record Holder

### Micro MBA 
**Rady School of Management, University of California, San Diego**
*May 2023 - September 2023*

## Technical Skills
Python ० Jupyter Lab ० Linux Command Line ० Github ० R ० RStudio ० Slurm Workload Manager ०
Shell Scripting ० Anaconda ० Amazon Web Services ० Microsoft Office ० Adobe Illustrator ० Vim ० C++

## Professional Expereince
- Improved computational pipelines for the detection of cfmDNA (cell-free microbial DNA)
- Used scikit-learn machine learning for the classification of microbial differences across cancer tissue
- Ran BWA, Bowtie2, Minimap2, Samtools, and other common bioinformatic command line tools
- Gathered and processed large genomic cohorts such as The Cancer Genome Atlas & genomes on NCBI
- Written clean, well commented, reproducible code with Github Repositories and Conda environments
- Processed over 100 terabytes of whole genome sequencing data using high performance computing
- Microbial analysis of 16S and shotgun datasets including alpha and beta diversity metrics such as PCoA

## Leadership & Mentorship
- Project manager for the filtering, aggregation & analysis of over 150 cancer microbiome datasets (13)
- Independently supervised Ph.D. Visiting Scholar and two undergraduate students
- Project deliverables included drafted publication and three invited conference presentation
- Graduate Bioinformatics Council: Community Event Chair + Outreach Committee Member
- NCAA Division I Student-Athlete & Captain

## Presentations
- “Improving microbial detection in cancer tissue samples with computational host depletion using the
pangenome”, Poster at American Association for Cancer Research Annual Meeting, April 2024
- “Enhancing microbial insights in cancerous tissue: unveiling the hidden microbiome” Cancer
Genomics Cloud, Seminar Series; July 2023
- “Evolutionary modeling of microbiome community assembly in the context of human pre-cancer
progression.” Talk at Math Modeling of Microbes Conference, Max Planck Institute;September 2022
- “Community assembly dynamics of the microbiome in premalignant Barrett’s esophagus.” Structural &
Functional Genomics Retreat, Moores Cancer Center; March 2022

## Awards
- Voted Best Presentation - UC San Diego Bioinformatics Program Conference, 2024
- Winifred B. Keaney Award: Role model student-athlete - Univ. of Rhode Island, 2020
- Biology Department Alpha Research Award - Univ. of Rhode Island, 2020
- Winifred P. Kelley Award of Academic Excellence - Univ. of Rhode Island, 2017

-----

## Experience
### Bioinformatics Graduate Reseracher 
**University of California, San Diego**  
*September 2020 - September 2024 (predicted)*
- Designed computational host filtration pipeline to remove human reads for microbial analysis (10)
- Applied math modeling to determine microbial evolution across esophageal cancer development (12)
- Computationally extracted microbial reads from blood and determined microbial differences across
liver disease (11) using differential abundance (6) and random forest machine learning (9)
- **Invited Talk**: [Cancer Microbiome](https://youtu.be/a7VBn42wyjo?si=sStP3vGwFq5o_pPs)

### Undergraduate Reserach Assistant
**University of Rhode Island**  
*January 2018 - September 2020*
- **Adviosr** : [Dr. Rachel Schwartz](https://schwartzlaburi.github.io)
- Performed machine learning on phylogenetic markers to identify which traits lead to speciation
- **Poster presentation** : [Rethinking the Evolutionary Tree of Life using Machine Learning](https://digitalcommons.uri.edu/srhonorsprog/758/)

### Informatics Intern
**[Agios Pharmaceuticals](https://www.agios.com)**  
*Summer 2019*
- Implemented a Laboratory Information Management System for reproducible research using R

### Discrete Mathematics & Theoretical Computer Science Research Experience for Undergraduates
**[Rutgers Cancer Institute of New Jersey](https://www.cinj.org)**  
*Summer 2018*
- **Adviosr** : [Dr. Hossein Khiabanian](http://www.khiabanian-lab.org/pages/team.html)
- Implemented a program that estimated purity reading for cancer tissue samples using Python (1)
- **Project website** : [Elucidating Tumor Evolutionary Patterns Using High-Depth Molecular Data](http://reu.dimacs.rutgers.edu/~cguccion/)

### Data-Intensive Scientific Computing Research Experience for Undergraduates 
**University of Notre Dame**  
*Summer 2018*
- **Adviosr** : [Dr. Douglas Thain](https://www3.nd.edu/~dthain/)
- Created workflows to allow non-computational users to run BWA on Amazon Web Services
- **Project video** : [Simplifying Bioinformatics Workflows](https://www.youtube.com/watch?v=o0AVWyQJGCE)

-----

## Publications
[13] **Guccione, C.#**, Takaces, B.#, et al. Comparative Analysis of the Microbial Composition of Different Cancer
Tumor Tissues [In preparation for June 2024 submission] 
  
[12] **Guccione, C.**, et al. Community assembly modeling of microbial evolution within Barrett’s esophagus and
esophageal adenocarcinoma. [In preparation for ISME J March 2024 submission]  
  
[11] **Guccione, C.#**, Dantas Machado, A.#, et al. Microbial DNA in tumor and blood differentiates primary and
secondary hepatic malignancies [In preparation for Cell & Host Microbiome April 2024 submission]  
  
10 **Guccione, C.#**, Patel, L.#, et al. Incomplete human reference genomes can drive false sex biases and expose
patient-identifying human sequences in metagenomic data. [Submitted to Nature Microbiology]
  
9 **Guccione, C.#**, Sepich-Poore, G.#, McDonald, D.#, Evguenia, K.#, et al., Robustness of cancer microbiome
signals over a broad range of methodological variation. Oncogene (2024).
  
8 Chapman, O. S., Luebeck, J., Sridhar, S., et al. Circular extrachromosomal DNA promotes tumor
heterogeneity in high-risk medulloblastoma. Nature Genetics (2023).
  
7 **Guccione, C.**, McDonald, D., et al.. You are what you excrete. Nature Microbiology (2023).
  
6 Rahman, G., Morton, J., Martino, C., Sepich-Poore, G., Allaband, **C., Guccione**, C., et al. BIRDMAn: A
Bayesian differential abundance framework that enables robust inference of host-microbe associations.
bioRxiv (2023). [Submitted to Nature Microbiology]
  
5 Ghosh, P., Campos, V., Vo, D., **Guccione, C.**, et al. AI-assisted discovery of an ethnicity-influenced driver of
cell transformation in esophageal & gastroesophageal junction adenocarcinomas. JCI Insight (2022).
  
4 Ma, S. D., **Guccione, C.**, et al. Oral Microbiome Diversity Between Patients With Normal Versus Elevated
Salivary Pepsin Levels & Its Implication in Gastroesophageal Reflux Disease. Foregut (2022).
  
3 **Guccione, C.#**, Sepich-Poore, G. D.#, et al. Cancer's second genome: Microbial cancer diagnostics and
redefining clonal evolution as a multispecies process. BioEssays (2022).
  
2 **Guccione, C.**, Yadlapati, R., et al. Challenges in Determining the Role of Microbiome Evolution in Barrett’s
Esophagus and Progression to Esophageal Adenocarcinoma. Microorganisms (2021).
  
1 Loh, J. W., **Guccione, C.**, et al. All-FIT: Allele-frequency-based imputation of tumor purity from high-depth
sequencing data. Bioinformatics (2020).

#Authors contributed equally, [ ] In preparation for submission
